Migration.Tool.Extensions/README.md

```diff
@@ -1,123 +1,78 @@
-  - implement method `ShallMigrate`. If method returns true, migration will be used. This method receives a context, by which you can decide - typically by the original widget's type
-  - implement `MigrateWidget`, where objective is to convert old JToken representing the widget's JSON to new converted JToken value
-    - Widget property migration will still be applied after your custom widget migration
-    - In the following cases, you must explicitly specify the property migration to be used, via `PropertyMigrations` in returned value (because it can't be infered from the original widget)
-      - If you add a new property. That includes renaming an original property.
-      - In the special case when you introduce a new property whose name overlaps with original property. Otherwise the migration infered from the original property would be used
-        - If your new property is not supposed to be subject to property migrations and the original one was, explicitly specify `WidgetNoOpMigration` for this property
-    - You can also override the property migration of an original property if that suits your case
-    
-- finally register in `Migration.Tool.Extensions/ServiceCollectionExtensions.cs` as `Transient` dependency into service collection. For example `services.AddTransient<IWidgetMigration, YourMigrationClass>()`
-
-Samples:
-- [Sample BannerWidget migration](./CommunityMigrations/SampleWidgetMigration.cs)
-
-### Convert page type to reusable content item (content hub)
-
-demonstrated in method `AddReusableRemodelingSample`. Please note, that all information unique to page will be lost
-
-## Custom widget property migrations
-
-To create custom widget property migration:
-- create new file in `Migration.Tool.Extensions/CommunityMigrations` (directory if you need more files for single migration)
-- implement interface `Migration.Tool.KXP.Api.Services.CmsClass.IWidgetPropertyMigration`
-  - implement property `Rank`, set number bellow 100 000 - for example 5000. Rank determines the order by which the migrations are tested to be eligible via the `ShallMigrate` method
-  - implement method `ShallMigrate` (if method returns true, migration will be used)
-  - implement `MigrateWidgetProperty`, where objective is to convert old JToken representing json value to new converted JToken value  
-- finally register in `Migration.Tool.Extensions/ServiceCollectionExtensions.cs` as `Transient` dependency into service collection. For example `services.AddTransient<IWidgetPropertyMigration, WidgetPathSelectorMigration>()`
-
-Samples:
+# Migration Tool customization
+
+To create custom migrations:
+
+1. Create the custom migration class:
+    - [Field migrations](#Customize field migrations)
+    - [Widget migrations](#Customize widget migrations)
+    - [Widget property migrations](#Customize widget property migrations)
```

Comment:
Do these links really work? I am nor sure whether this is the correct format of the section link 
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links